src/models/thelio-major-r3/internal-overview.md

```diff
@@ -0,0 +1,41 @@
+# Thelio Major (Internal Overview)
+
+## Left side overview:
+
+![Internal overview (left side)](./img/components-left.webp)
+
+- CPU cooler & CPU are highlighted in red (located behind the CPU shroud)
+- RAM slots are highlighted in green (located behind the CPU shroud)
+- 2.5" SATA drive cage (with cover removed) is highlighted in pink
+- M.2 slots are highlighted in white (located underneath the GPU & heatsinks)
+- GPU (in PCIe 5.0 x16 slot) is highlighted in yellow
+- PCIe 4.0 x8 and 3.0 x2 slots are highlighted in navy
+- Power supply is highlighted in cyan
+- Case fans are highlighted in orange
+
+## Right side overview:
+
+![Internal overview (right side)](./img/components-right.webp)
+
+- Thelio-IO access cutout is highlighted in green
+- Power supply access cutout is highlighted in red
+
+## Front side overview:
+
+![Internal overview (front side)](./img/components-front.webp)
+
+- Thelio-IO access cutout is highlighted in green
+
+## Top overview:
+
+![Internal overview (top)](./img/components-top.webp)
+
+- 2.5" drive screws (with retainer ring removed) are highlighted in green
+- CPU power connectors are highlighted in red
+    - Connect to power supply
```

Comment:
s/Connect/Connects

Reply:
I put "connect" because there are two "connectors," plural. Do you think I should go back to calling it a single connector despite having two plugs? (I was on the fence about this.)

Reply:
Ah, I see. Now I'm on the fence about it as well, but now I think "connect" is fine. I'll mark this as resolved, but I'll leave the decision up to you. It just stood out to me since the other lines say "connect", and I had a reflexive assumption that it was a typo.